### ISSUE TITLE
--scope does not work anymore

### ISSUE THREAD

**Issue by StarpTech** [lerna/lerna#1556]
## Expected Behavior
I can select a specific package to publish with `--scope`. It worked in 2.X

## Current Behavior
```
ERR! lerna Unknown argument: scope
```

## Your Environment

| Executable | Version |
| ---: | :--- |
| `lerna --version` | 3.0.1 |
| `npm --version`  | 5.6.0 |
| `node --version` | 9.11.2 |

| OS | Version |
| --- | --- |
| NAME | VERSION |
| Windows 10 | 1607 |

**Comment by evocateur** [comment]
If you always want to publish a specific package,  you probably shouldn't be using lerna. You can always descend into the directory and call `npm publish` yourself.

In any case, `--scope` applied to `lerna publish` was always a bug, as it breaks fundamental assumptions of change detection. v3 merely fixed the bug.

**Comment by StarpTech** [comment]
I think you didn't understand me. I use the independent version mode. **Sometimes** I want to publish only one package. When I do `npm publish` all my settings from the `lerna publish` command are lost and lerna does not upgrade the version inside the dependent packages.

**Comment by evocateur** [comment]
This is the fundamental trade off of a monorepo: when using git tags to detect changed packages, independently or not, you need to be okay with publishing _whatever_ is "ready" at any given time. (This should already be the case if we're talking about the `master` branch)

**Comment by StarpTech** [comment]
OK, that's fine but then only updated packages should be listed right? I will try to reproduce but my feeling is that this doesn't work reliable.

**Comment by jbailey4** [comment]
@StarpTech Issue #1544 might be of some help, good discussion over there on the `--since` flag.

tl;dr `lerna ls` still accepts the `--since` flag to print out packages changed since a given revision.
